File: 2440.c
#include<stdio.h>

int main(){
    int input = 0;
    scanf("%d", &input);
    
    for(input; input > 0; input--){
        for(int i = 1; input >= i; i++)
            printf("*");
        printf("\n");
    }
    return 0;
}

File: 2440.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int input;
        
        for(input = sc.nextInt(); input > 0; input--){
            for(int i = 1; input >= i; i++)
                System.out.print("*");
            System.out.println();
        }
    }
}
